### ISSUE TITLE
Error creating new store

### ISSUE THREAD

**Issue by arsfutura** [solidusio/solidus_multi_domain#18]
Greetings,

I'm getting a weird error with this gem. 

When I try to create or edit new store in Settings -> Stores & Domains (in development) I get the following error:

```
undefined method `payment_methods' for #<Spree::Store:0x007fd07c1f0500>
```
``` 
else
        match = match_attribute_method?(method.to_s)
        match ? attribute_missing(match, *args, &block) : super
      end
end
```

Did someone experience this, can it be some kind conflict with other gems?

**Comment by athal7** [comment]
@arsfutura you may need to update your version of Solidus, which version are you currently using?

**Comment by athal7** [comment]
I've just released 1.0.3 to rubygems which should fix your issue if you are on Solidus 1.0.x. Feel free to use 1.1.0 if you are on Solidus 1.1.0.pre

**Comment by arsfutura** [comment]
Now it works correctly, thank you.

Additionally I needed to update solidus to 1.0.1 version (it was 1.0.1) and make migrations install because I was missing table spree_store_payment_methods.
